operations/services.md

```diff
@@ -99,8 +99,10 @@ Returns all services offered by the enterprise.
 | `OccupancyEndOffset` | string | required | Offset from the end of the time unit defining the occupancy end of the service in ISO 8601 duration format that is considered regarding the availability and reporting. |
 | `TimeUnit` | [Time unit period](#time-unit-period) | required | Time unit period of the service. |
 
+
+#### Time unit start and end
 Time units represent a fixed, finite time interval: a minute, a day, a month, etc. A Time unit defines the operable periods for a bookable service.
-We think of the daily time unit as the physical time unit that starts at midnight and ends at midnight the following day. The monthly time unit is time unit that starts at mightnight of the first day of the month and ends at midnight of the first day of the following month.
+We think of the daily time unit as the physical time unit that starts at midnight and ends at midnight the following day. A monthly time unit is a time unit that starts at midnight on the first day of the month and ends at midnight on the first day of the following month.
```

Comment:
> We think of the daily time unit as the physical time unit that starts at midnight and ends at midnight the following day.

I'm wondering if there's a different way to describe this? I had understood that midnight the following day means the start of the next day, so if I only wanted to indicate one daily time unit via the API, I should send the same Start/End time, or End time any time before midnight the next day. As a partner reading the documentation I might not know this and assume that sending midnight today to midnight the next day would indicate one daily time unit.

Reply:
It should not longer be the case with the new API that drops StartUtc and EndUtc and work with FirstTimeUniT and LastTimeUnit no?
This paragraph is about describing what is the time unit. Then there is the picture that is aligned with what is described here.
From partner we then want to send StartUtc of the FirstTimeUnit they would like to affect and StartUtc of the LastTimeUnit. Do you think it is not understandable that for updating the one day the FirstTimeUnit and LastTimeUnit would be the same time unit? (with the same StartUtc).